src/main/webapp/script.js

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+window.addEventListener('load', () => {
+  let selected = [];  // Courses selected by the user
+  let courses = [];   // List with all courses
+
+  getOptions();
+
+  document.getElementById('add-selected').addEventListener('click', function() {
```

Comment:
let's be consistent and use the arrow style functions since you're doing it elsewhere:

```
.addEventListener('click', () => {
  addToSelected();
});
```